### ISSUE TITLE
Doesn't work in web `textDocument/documentColor`

### ISSUE THREAD

**Issue by zardoy** [johnsoncodehk/volar#1479]
Hey! I really want to use document features in the browser, but it doesn't seem to work.

To reproduce:

1. Open fresh vscode.dev
2. Install volar
3. Create new untitled vue file
4. It starts spamming that `Request textDocument/documentColor failed.`

I also tried this file:

```vue
<template>
<div class="test"></div>
</template>
<script>
export default {
    methods() {
        sayHi() {}
    }
}
</script>
```

it doesn't give any outline.

I'm creating this issue, since https://github.com/johnsoncodehk/volar/issues/612 says that document features are supported

**Comment by johnsoncodehk** [comment]
It seems a regression, thank to report!

**Comment by johnsoncodehk** [comment]
Still not working in v0.40.2 because due to bundle issue, it will be fix in v0.40.3.
